ixnetwork_open_traffic_generator/ixnetworkapi.py

```diff
@@ -261,7 +261,7 @@ def _connect(self):
         """Connect to an IxNetwork API Server.
         """
         if self._assistant is None:
-            platform = TestPlatform(self._address, rest_port=self._password)
+            platform = TestPlatform(self._address, rest_port=self._port)
```

Comment:
Is it relevant to this PR ? if not, please include this as part of separate PR (linked to an issue).

Reply:
raise one more PR for it
https://github.com/open-traffic-generator/ixnetwork/pull/185/commits/a1ed8dc03b51b1e40bd80ac3738fbb4c0efc5a4f